### PR TITLE
Form validator

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
here are the things I added to the repository
-create a new branch called form validator 
add a new element to Html to handle the error message so the user could see that their input was not correct
add new file called form.js 
write the logic on the form.js file 
the function inside the form.js file is to handle the error message if the user typed a wrong format on the contact form

### CODE REVIEW

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi **@AlaaAlsalem** , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't contain the `eslint settings` in the `.github/workflows/linters.yml` directory.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (SajjadAhmad14) [inline .eslintrc.json:1]: Please add code linter for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) config.

File: index.html [hunk 2]
```diff
@@ -229,18 +244,20 @@ <h1>About Me</h1>
         </section>
 
         </section>
-        <section class="contact_form">
```
Comment (SajjadAhmad14) [inline index.html:232]: Please place the error message close to the "submit" button.

File: index.html [hunk 3]
```diff
@@ -14,25 +14,40 @@
...
+                <li class=" headbtn mob_hdn d_view"><a href="#">Portfolio</a></li>
+                <li class=" headbtn mob_hdn d_view"> <a href="#">About</a></li>
+                <li class="headbtn mob_hdn d_view"> <a href="#">contact</a></li>
+                <li class="headbtn mob_hdn d_view"><i class="fa fa-envelope" aria-hidden="true"></i></li>
                 <li id="nav-btn"><nav class="fas fa-bars hamburger-icon menu"></nav></li>
             </ul>
+           
         </nav>
+       
     </header>
```
Comment (SajjadAhmad14) [inline index.html:43]: Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and apply best practices to write professional commit messages.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Aladdin Alsalem, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by mke2111 — APPROVED** [review]
### Hi @AlaaAlsalem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**